it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/ruleeditor/submit/submitdefaulterrorhandler/.content.xml

```diff
@@ -3,20 +3,28 @@
     jcr:primaryType="cq:Page">
     <jcr:content
         cq:deviceGroups="[/etc/mobile/groups/responsive]"
-        cq:lastModified="{Date}2024-04-10T06:59:23.294Z"
+        cq:lastModified="{Date}2024-06-25T16:30:36.396Z"
         cq:lastModifiedBy="admin"
         cq:template="/conf/core-components-examples/settings/wcm/templates/af-blank-v2"
         jcr:language="en"
         jcr:primaryType="cq:PageContent"
         jcr:title="submitDefaultErrorHandler"
         sling:configRef="/conf/forms/core-components-it/samples/ruleeditor/submit/submitdefaulterrorhandler/"
-        sling:resourceType="forms-components-examples/components/page">
+        sling:resourceType="forms-components-examples/components/page"
+        ignoreTranslationInvalidation="{Boolean}true">
```

Comment:
What is this flag ? Can you elaborate on the use-case ?

Reply:
It is OOTB. I just downloaded the form after changing submission to reset endpoint.